pkg/services/aws_tags.go

```diff
@@ -1,4 +1,4 @@
-package exporter
+package services
```

Comment:
This file feels a little out of place here since it's not helping define supported services like services.go. `TaggedResource` and `FilterThroughTags` is used by services but I I'm not sure the services pkg owns the definition of a `TaggedResource`. I think it might fit better with the `job` pkg, WDYT?